.eslintrc

```diff
@@ -10,15 +10,20 @@
     "react/prefer-es6-class": 0,
     "react/prefer-stateless-function": 0,
     "react/no-multi-comp": 0,
-    "array-callback-return": "off", 
+    "array-callback-return": "off",
     "arrow-body-style": ["warn", "as-needed"],
-    "comma-dangle": ["warn", "never"],
+    "comma-dangle": ["warn", "always"],
     "import/extensions": "off",
-    "import/no-unresolved": "off",
+    "import/no-unresolved": 1,
```

Comment:
This was off I think because it was erroring on coffeescript common js requires.